examples/examples_heterogeneous/001_heterogeneous_inflow_single.py

```diff
@@ -65,15 +65,20 @@
     x_resolution=200, y_resolution=100, height=90.0
 )
 
-# Plot the horizontal plane
+# Plot the horizontal plane using the visualize_heterogeneous_cut_plane
+# note this function is not very different than the standard visualize_cut_plane
+# accept that it accepts the fmodel object in order to visualize the boundary
+# of the heterogeneous inflow region.
 fig, ax = plt.subplots()
-visualize_cut_plane(
+visualize_heterogeneous_cut_plane(
     horizontal_plane,
+    fmodel=fmodel,
```

Comment:
That is true, I think originally it was this way and I just kept it.  Thinking it over now, the advantage of fmodel is that if you have a cut_plane, you must have an fmodel.  While it is not certain whether you have a heterogeneous_inflow_config,  heterogeneous_inflow_config_by_wd or HeterogeneousMap since at this top level theoretically you can define any, so maybe this is most simple approach?

Reply:
Ok, understood, sounds good.